frontend/src/assets/sass/app.scss

```diff
@@ -28,6 +28,17 @@ legend {
 
   }
 
+  // convenient spacing classes for checkboxes/radios that have an htmlLabel - the effect
+  // of putting a class='tw-space-below-4' on something inside the htmlLabel of an option
+  // is equivalent to putting 'mb-4' on the option div itself.
+  @each $size, $length in $spacers {
+    @each $dir, $edge in (above:top, below:bottom) {
+      :is(.custom-checkbox, .custom-radio):has(.tw-space-#{$dir}-#{$size}) {
```

Comment:
This makes use of the `:has` selector (supported in recent - i.e. 2024 - browsers) - I've introduced classes that you can put on an element inside the `htmlLabel` to influence the margins of the _containing_ `div` for that option.